src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Compiler/PropertyInfoPassTest.php

```diff
@@ -69,4 +71,24 @@ public function testReturningEmptyArrayWhenNoService()
 
         $this->assertEquals(array(), $actual);
     }
+
+    public function testRegisterSerializerExtractor()
+    {
+        $serializerExtractorDefinitionProphecy = $this->prophesize('Symfony\Component\DependencyInjection\Definition');
+        $serializerExtractorDefinitionProphecy->addArgument(Argument::type('Symfony\Component\DependencyInjection\Reference'))->shouldBeCalled();
+        $serializerExtractorDefinitionProphecy->addTag('property_info.list_extractor', array('priority' => -999))->shouldBeCalled();
+
+        $propertyInfoDefinitionProphecy = $this->prophesize('Symfony\Component\DependencyInjection\Definition');
+
+        $containerProphecy = $this->prophesize('Symfony\Component\DependencyInjection\ContainerBuilder');
+        $containerProphecy->hasDefinition('property_info')->willReturn(true)->shouldBeCalled();
```

Comment:
There's no need to check if a stub was called (which will make it a mock). It makes refactoring harder. Most of the time only one of `willReturn()` or `shouldBeCalled()` should be used, to either stub or mock.